src/ordered_set_lang.ml

```diff
@@ -141,6 +147,7 @@ module Make(Key : Key)(Value : Value with type key = Key.t) = struct
   end
 
   module Make(M : Named_values) = struct
+    [@@@warning "-56"] (* XXX necessary for 4.02.3 *)
```

Comment:
FYI I just suggested mnemonic aliases for warning codes on mantis: https://caml.inria.fr/mantis/view.php?id=7823

Reply:
That would help indeed :)

Reply:
My comment is a bit inaccurate I guess. You will get warning 56 for 4.06.1:

```
File "src/ordered_set_lang.ml", line 166, characters 10-19:
Error (warning 56): this match case is unreachable.
Consider replacing it with a refutation case '<pat> -> .'
```

But we cannot follow the advice of this warning because it wouldn't work in 4.02.3

Reply:
I see, thanks for the explanation